src/test/java/org/galatea/starter/entrypoint/IexRestControllerTest.java

```diff
@@ -75,10 +76,34 @@ public void testGetLastTradedPriceEmpty() throws Exception {
 
     MvcResult result = this.mvc.perform(
         org.springframework.test.web.servlet.request.MockMvcRequestBuilders
-            .get("/iex/lastTradedPrice?symbols=")
+            .get("/iex/lastTradedPrice?token=TEST&symbols=")
             .accept(MediaType.APPLICATION_JSON_VALUE))
         .andExpect(status().isOk())
         .andExpect(jsonPath("$", is(Collections.emptyList())))
         .andReturn();
   }
+
+  @Test
```

Comment:
I'd like to see more test cases. What happens when no parameters are passed? What happens when an invalid parameters are passed? What happens when the IEX server is down?